azure_media_services/static/js/player.js

```diff
@@ -2,19 +2,8 @@
 // Licensed under the MIT license. See LICENSE file on the project webpage for details.
 
 function AzureMediaServicesBlock(runtime, element) {
-  //
-  // IMPORTANT: We need to send in the DOM element which is the
```

Comment:
this long-winded comment is no longer applicable. the underlying issues have long-since been resolved by azure media player. we can now simply provide the video element id (as per their own spec). this fixes most of the bugs mentioned in this pull request description